FFCDemoPaymentService/Scheduling/CreateSchedule.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using FFCDemoPaymentService.Models;
+using FFCDemoPaymentService.Data;
+using Amqp;
+
+namespace FFCDemoPaymentService.Scheduling
+{
+    class CreateSchedule : IScheduling
+    {
+        private readonly ApplicationDbContext _db;
```

Comment:
To be consistent with the rest of the service don't use "_" as prefixes to private variables.

It's not against any C# standard but it's better if we do the same thing in every class.